tests/test_unittest/test_gtc/test_oir.py

```diff
@@ -30,3 +40,36 @@ def test_no_horizontal_offset_allowed():
 def test_mask_must_be_bool():
     with pytest.raises(ValidationError, match=r".*must be.* bool.*"):
         MaskStmtFactory(mask=FieldAccessFactory(dtype=DataType.INT32))
+
+
+def test_temporary_dimensions():
+    temp1d = Temporary(name="b", dtype=DataType.INT64, dimensions=(True, False, False))
+    assert temp1d.dimensions == (True, False, False)
```

Comment:
Technical point: this temporary is technically unable to be assigned to since not all the parallel axes are present. That probably doesn't matter here, but it might confuse readers down the road.

Reply:
You are right. I think this test is pretty useless and I will remove it